crates/bifrost/src/providers/replicated_loglet/replication/checker.rs

```diff
@@ -259,7 +273,7 @@ impl<Attr: Eq + Hash + Clone + std::fmt::Debug> NodeSetChecker<Attr> {
             .get(&node_id)
             .copied()
             .unwrap_or(StorageState::Disabled);
-        if !storage_state.should_read_from() {
+        if storage_state.empty() {
```

Comment:
Is there value in making a distinction between `Disabled` and `empty` with the latest changes? Maybe those two concepts could be unified?

Reply:
They can be unified, I just wanted to leave the door open for future state that can allow for active data rebuilding, in those cases, empty will not be equal to disabled.